components/dialog/stories/dialog.test.js

```diff
@@ -2,18 +2,49 @@ import { Variants } from "@spectrum-css/preview/decorators";
 import { Template } from "./template.js";
 
 export const DialogGroup = Variants({
-	Template,
+	Template: (args, context) => (
+		/*
+		 * This template forces each test case to showModal: false, and give a background 
+		 * color to the dialog grid container when isChromatic() is true.
+		 * This should ensure that the Sizes within the Variants() template to display correctly.
+		 */
+		window.isChromatic() ?
+			Template({
+				...args,
+				showModal: false,
+
+				// TODO: remove this heading arg when the "Sizing" heading is no longer overwritten by it.
+				heading: "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+				customStyles: {
+					"background-color": "var(--spectrum-gray-100)",
+				},
+			}, context)
+			: Template(args, context)
```

Comment:
It helps React's hot module reloader if we only call a template once and toggle the inputs based on settings:

```suggestion
		Template({
			...args,
			showModal: window.isChromatic() ? false : args.showModal,

			// TODO: remove this heading arg when the "Sizing" heading is no longer overwritten by it.
			heading: window.isChromatic() ? "Lorem ipsum dolor sit amet, consectetur adipiscing elit" : args.heading,
			customStyles: {
			    ...(args.customStyles ?? {}),
				"background-color": window.isChromatic() ? "var(--spectrum-gray-100)" : undefined,
			},
		}, context)
```

Reply:
ooooo awesome. Thank you!